guidelines.md

```diff
@@ -0,0 +1,28 @@
+# Guidelines
+## Writing
+
+When writing for ipfs, be it commit messages or posting issues, here's a collection of things to take note of.
+
+### Ungendering
+
+#### Motivation
+
+To avoid gendering in community interaction has multiple benefits. Avoiding the alienating feeling of documentation implicitly misgendering you, "the user and his computer". This is true of roughly half the population. Using plural forms is a good hack on the english language to get around this problem. This also includes extraterrestial beings and future lifeforms.
```

Comment:
Not sure this gets the point across. How about:

Avoiding gendering in community interaction has multiple benefits. Using a male or female pronoun alienates and unfairly prioritizes one gender in favor of another. For instance, saying "the user and his computer" may feel alienating to women, or vice versa. Luckily, in English, the pronoun 'their' can be used as both a singular and a plural form (cf. [Singular they](https://en.wikipedia.org/wiki/Singular_they)). Saying, "the user and their computer" is entirely acceptable and alleviates the issue. Furthermore, this also includes extraterrestial beings and future life forms.